manifests/j/JTechMe/JumpGoBrowser/Legacy/4.4/JTechMe.JumpGoBrowser.Legacy.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: JTechMe.JumpGoBrowser.Legacy
+PackageVersion: 4.4
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/JTechMe/JumpGo-for-Windows/releases/download/v4.4/JGSetup.msi
+  InstallerSha256: A5431CC3E76A9154CEB862ED84D63538EA3860B88AF3541B98DEC2F39EE0D746
+  InstallModes:
+  - interactive
+  ProductCode: '{F054283B-825D-4DFA-8F6A-AE36D59706D3}'
+  UnsupportedOSArchitectures:
+  - arm
+  - arm64
+  AppsAndFeaturesEntries:
+  - DisplayName: JumpGo Browser
+    Publisher: JTechMe
+    DisplayVersion: 4.4
+    UpgradeCode: '{53B765DB-78E3-434E-8014-CBF05A89CAB6}'
+  ElevationRequirement: elevatesSelf
+  ReleaseDate: 08-03-2016

```

```suggestion
  ReleaseDate: 2016-08-03
```
needs to follow ISO 8601 format